Traceback (most recent call last):
  File "//anaconda3/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "//anaconda3/lib/python3.10/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "//anaconda3/lib/python3.10/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "//anaconda3/lib/python3.10/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "//anaconda3/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "//anaconda3/lib/python3.10/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "//anaconda3/lib/python3.10/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "//anaconda3/lib/python3.10/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# create dataframe from counts of "Trust_Index" values
levels = wgm["Trust_Index"].value_counts().reset_index(name="Count").rename(columns={"index": "Trust_Level"})

# create bar plot
ax = sns.barplot(
    data = levels,
    x = "Trust_Level",
    y = "Count",
    order = ["No score", "Low trust", "Medium trust", "High trust"]
)

# label each bar with the corresponding value
for container in ax.containers:
    ax.bar_label(container)

# set axis labels and title
ax.set_xlabel("")
ax.set_title("Trust in Scientists Index, Worldwide");
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mValueError[0m                                Traceback (most recent call last)
Cell [0;32mIn[4], line 5[0m
[1;32m      2[0m levels [38;5;241m=[39m wgm[[38;5;124m"[39m[38;5;124mTrust_Index[39m[38;5;124m"[39m][38;5;241m.[39mvalue_counts()[38;5;241m.[39mreset_index(name[38;5;241m=[39m[38;5;124m"[39m[38;5;124mCount[39m[38;5;124m"[39m)[38;5;241m.[39mrename(columns[38;5;241m=[39m{[38;5;124m"[39m[38;5;124mindex[39m[38;5;124m"[39m: [38;5;124m"[39m[38;5;124mTrust_Level[39m[38;5;124m"[39m})
[1;32m      4[0m [38;5;66;03m# create bar plot[39;00m
[0;32m----> 5[0m ax [38;5;241m=[39m [43msns[49m[38;5;241;43m.[39;49m[43mbarplot[49m[43m([49m
[1;32m      6[0m [43m    [49m[43mdata[49m[43m [49m[38;5;241;43m=[39;49m[43m [49m[43mlevels[49m[43m,[49m
[1;32m      7[0m [43m    [49m[43mx[49m[43m [49m[38;5;241;43m=[39;49m[43m [49m[38;5;124;43m"[39;49m[38;5;124;43mTrust_Level[39;49m[38;5;124;43m"[39;49m[43m,[49m
[1;32m      8[0m [43m    [49m[43my[49m[43m [49m[38;5;241;43m=[39;49m[43m [49m[38;5;124;43m"[39;49m[38;5;124;43mCount[39;49m[38;5;124;43m"[39;49m[43m,[49m
[1;32m      9[0m [43m    [49m[43morder[49m[43m [49m[38;5;241;43m=[39;49m[43m [49m[43m[[49m[38;5;124;43m"[39;49m[38;5;124;43mNo score[39;49m[38;5;124;43m"[39;49m[43m,[49m[43m [49m[38;5;124;43m"[39;49m[38;5;124;43mLow trust[39;49m[38;5;124;43m"[39;49m[43m,[49m[43m [49m[38;5;124;43m"[39;49m[38;5;124;43mMedium trust[39;49m[38;5;124;43m"[39;49m[43m,[49m[43m [49m[38;5;124;43m"[39;49m[38;5;124;43mHigh trust[39;49m[38;5;124;43m"[39;49m[43m][49m
[1;32m     10[0m [43m)[49m
[1;32m     12[0m [38;5;66;03m# label each bar with the corresponding value[39;00m
[1;32m     13[0m [38;5;28;01mfor[39;00m container [38;5;129;01min[39;00m ax[38;5;241m.[39mcontainers:

File [0;32m//anaconda3/lib/python3.10/site-packages/seaborn/categorical.py:2755[0m, in [0;36mbarplot[0;34m(data, x, y, hue, order, hue_order, estimator, errorbar, n_boot, units, seed, orient, color, palette, saturation, width, errcolor, errwidth, capsize, dodge, ci, ax, **kwargs)[0m
[1;32m   2752[0m [38;5;28;01mif[39;00m estimator [38;5;129;01mis[39;00m [38;5;28mlen[39m:
[1;32m   2753[0m     estimator [38;5;241m=[39m [38;5;124m"[39m[38;5;124msize[39m[38;5;124m"[39m
[0;32m-> 2755[0m plotter [38;5;241m=[39m [43m_BarPlotter[49m[43m([49m[43mx[49m[43m,[49m[43m [49m[43my[49m[43m,[49m[43m [49m[43mhue[49m[43m,[49m[43m [49m[43mdata[49m[43m,[49m[43m [49m[43morder[49m[43m,[49m[43m [49m[43mhue_order[49m[43m,[49m
[1;32m   2756[0m [43m                      [49m[43mestimator[49m[43m,[49m[43m [49m[43merrorbar[49m[43m,[49m[43m [49m[43mn_boot[49m[43m,[49m[43m [49m[43munits[49m[43m,[49m[43m [49m[43mseed[49m[43m,[49m
[1;32m   2757[0m [43m                      [49m[43morient[49m[43m,[49m[43m [49m[43mcolor[49m[43m,[49m[43m [49m[43mpalette[49m[43m,[49m[43m [49m[43msaturation[49m[43m,[49m
[1;32m   2758[0m [43m                      [49m[43mwidth[49m[43m,[49m[43m [49m[43merrcolor[49m[43m,[49m[43m [49m[43merrwidth[49m[43m,[49m[43m [49m[43mcapsize[49m[43m,[49m[43m [49m[43mdodge[49m[43m)[49m
[1;32m   2760[0m [38;5;28;01mif[39;00m ax [38;5;129;01mis[39;00m [38;5;28;01mNone[39;00m:
[1;32m   2761[0m     ax [38;5;241m=[39m plt[38;5;241m.[39mgca()

File [0;32m//anaconda3/lib/python3.10/site-packages/seaborn/categorical.py:1530[0m, in [0;36m_BarPlotter.__init__[0;34m(self, x, y, hue, data, order, hue_order, estimator, errorbar, n_boot, units, seed, orient, color, palette, saturation, width, errcolor, errwidth, capsize, dodge)[0m
[1;32m   1525[0m [38;5;28;01mdef[39;00m [38;5;21m__init__[39m([38;5;28mself[39m, x, y, hue, data, order, hue_order,
[1;32m   1526[0m              estimator, errorbar, n_boot, units, seed,
[1;32m   1527[0m              orient, color, palette, saturation, width,
[1;32m   1528[0m              errcolor, errwidth, capsize, dodge):
[1;32m   1529[0m [38;5;250m    [39m[38;5;124;03m"""Initialize the plotter."""[39;00m
[0;32m-> 1530[0m     [38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43mestablish_variables[49m[43m([49m[43mx[49m[43m,[49m[43m [49m[43my[49m[43m,[49m[43m [49m[43mhue[49m[43m,[49m[43m [49m[43mdata[49m[43m,[49m[43m [49m[43morient[49m[43m,[49m
[1;32m   1531[0m [43m                             [49m[43morder[49m[43m,[49m[43m [49m[43mhue_order[49m[43m,[49m[43m [49m[43munits[49m[43m)[49m
[1;32m   1532[0m     [38;5;28mself[39m[38;5;241m.[39mestablish_colors(color, palette, saturation)
[1;32m   1533[0m     [38;5;28mself[39m[38;5;241m.[39mestimate_statistic(estimator, errorbar, n_boot, seed)

File [0;32m//anaconda3/lib/python3.10/site-packages/seaborn/categorical.py:541[0m, in [0;36m_CategoricalPlotter.establish_variables[0;34m(self, x, y, hue, data, orient, order, hue_order, units)[0m
[1;32m    539[0m     [38;5;28;01mif[39;00m [38;5;28misinstance[39m(var, [38;5;28mstr[39m):
[1;32m    540[0m         err [38;5;241m=[39m [38;5;124mf[39m[38;5;124m"[39m[38;5;124mCould not interpret input [39m[38;5;124m'[39m[38;5;132;01m{[39;00mvar[38;5;132;01m}[39;00m[38;5;124m'[39m[38;5;124m"[39m
[0;32m--> 541[0m         [38;5;28;01mraise[39;00m [38;5;167;01mValueError[39;00m(err)
[1;32m    543[0m [38;5;66;03m# Figure out the plotting orientation[39;00m
[1;32m    544[0m orient [38;5;241m=[39m infer_orient(
[1;32m    545[0m     x, y, orient, require_numeric[38;5;241m=[39m[38;5;28mself[39m[38;5;241m.[39mrequire_numeric
[1;32m    546[0m )

[0;31mValueError[0m: Could not interpret input 'Trust_Level'
ValueError: Could not interpret input 'Trust_Level'

